avm/res/app/container-app/main.bicep

```
@@ -595,3 +611,27 @@ type secretType = {
   @secure()
   value: string?
 }
+
+@export()
+@description('The type for the container app\'s authentication configuration.')
+@description('The type for the container app\'s authentication configuration.')
```

```suggestion
@description('The type for the container app\'s authentication configuration.')
```
one should be enough 😄